content/html/concepts/semantic-html/terms/section/section.md

```diff
@@ -61,3 +62,97 @@ Suppose the current page wants to add two sections:
   </body>
 </html>
 ```
+
+## Alternatives to `<section>`
```

Comment:
@D-Salkovic upon second look, we should turn these into bullets (without the html code block that follows) and link out to the respective entries on each element like:


There are also examples where we can use other elements that semantically fit better than the `<section>` element:
  
- The [`<header>`](https://www.codecademy.com/resources/docs/html/semantic-html/header) element which can be used to describe that the content inside is the header of the page.
- The [`<nav>`](https://www.codecademy.com/resources/docs/html/semantic-html/nav) element. It is used to describe that the content inside is a navigation system of the page.
- The [`<main>`](https://www.codecademy.com/resources/docs/html/semantic-html/main) element which is used to describe that the content inside is the main content of the page.
- The [`<footer>`](https://www.codecademy.com/resources/docs/html/semantic-html/footer) element which is used to describe that the content inside is the footer of the page.